src/test/burn/WixTestTools/RuntimeDomainFactAttribute.cs

```diff
@@ -0,0 +1,38 @@
+// Copyright (c) .NET Foundation and contributors. All rights reserved. Licensed under the Microsoft Reciprocal License. See LICENSE.TXT file in the project root for full license information.
+
+namespace WixTestTools
+{
+    using System;
+    using System.DirectoryServices.ActiveDirectory;
+    using System.Security.Principal;
+    using WixInternal.TestSupport.XunitExtensions;
+
+    public class RuntimeDomainFactAttribute : RuntimeFactAttribute
```

Comment:
Rather than create another `[Fact]` type, what about adding `RequiresDomain` bool property (or some better name) to the `RuntimeFact` attribute?

Reply:
done, because Named properties on attributes are only updated after the empty constructor has been called the code doesn't look very nice... but I think if using the same Attribute, having properly named attribute parameter(s) is better than just having nameless constructor parameters.